server/authentication.js

```diff
@@ -26,43 +25,42 @@ const checkJwt = expressjwt({
   getToken
 })
 
-const getUserInfo = async token => {
+const getUserInfo = token => {
   const config = {
     headers: {
       Accept: 'application/json',
       Authorization: `Bearer ${token}`
     }
   }
-  try {
-    return await axios.get('https://fog-uwa.au.auth0.com/userinfo', config)
-  } catch (error) {
-    return null
-  }
+  return axios.get('https://fog-uwa.au.auth0.com/userinfo', config)
 }
 
 const findUser = async req => {
   const token = getToken(req)
   if (!token) throw new Error('Error retrieving token')
-  const userInfo = await getUserInfo(token)
-  if (!userInfo) throw new Error('Error retrieving User Info')
-  const { email } = userInfo.data
-  const adminObject = await findUserByEmail(email)
-  return adminObject
+  try {
+    const userInfo = await getUserInfo(token)
+    const { email } = userInfo.data
+    const adminObject = await findUserByEmail(email)
+    const { name, _id } = adminObject
+    return { name, _id } || null
+  } catch (error) {
+    if(error.response.statusText === 'Unauthorized') throw new Error('User is unauthrorised')
+  }
 }
 
-const userAuthorised = async req => {
+const userIsAdmin = async req => {
   try {
     const adminObject = await findUser(req)
-    return Object.keys(adminObject).length !== 0
+    return !!adminObject
   } catch (error) {
```

Comment:
error is no longer used, isn't eslint throwing a fit?

Reply:
My eslint is cooked for this project